python/rmm/_lib/memory_resource.pyx

```diff
@@ -77,18 +77,24 @@ cdef extern from "rmm/mr/device/managed_memory_resource.hpp" \
 
 cdef extern from "rmm/mr/device/cuda_async_memory_resource.hpp" \
         namespace "rmm::mr" nogil:
-    cdef cppclass cuda_async_memory_resource(device_memory_resource):
-        enum allocation_handle_type:
-            none,
-            posix_file_descriptor,
-            win32,
-            win32_kmt
 
+    cdef cppclass cuda_async_memory_resource(device_memory_resource):
         cuda_async_memory_resource(
             optional[size_t] initial_pool_size,
             optional[size_t] release_threshold,
             optional[allocation_handle_type] export_handle_type) except +
 
+cdef extern from "rmm/mr/device/cuda_async_memory_resource.hpp" \
```

Comment:
I hate everything about  this, but it's the best  we can do until we use Cython 3.0 which has support for `enum class`.

Reply:
Added a TODO so that we clean this up when we get to Cython 3.0.